frontend/src/views/Game.tsx

```diff
@@ -63,7 +83,7 @@ const LeaderboardContent = styled.div`
     ${({ theme: { colors: { background } } }) => `linear-gradient(to right, rgba(0,0,0.8,.12), ${background})`},
     ${({ theme: { colors: { background } } }) => `linear-gradient(to left, rgba(0,0,0,.12), ${background})`};
 
-  background-position: left center, right center, left center, right center;
+  background-position: left center, right center, 0.15% center, 99.85% center;
   background-repeat: no-repeat;
   background-color: ${({ theme }) => theme.colors.background};
   background-size: 20px 100%, 20px 100%, 10px 100%, 10px 100%;
```

Comment:
image: the shadow linear gradients are the shadows you see on the leaderboard when there's scroll, whereas the shadow cover linear gradients are white (ish) shadows that cover those shadows when there's not scroll (there's 4 because of left and right + shadow and shadow cover
position: sets each of those four to be positioned at these places (left center would be 0% 50% i.e. left 0% and top 50%)
repeat: no idea, my guess is just makes it appear once rather than repeating itself
color: background color
size: for each of the four, set the width and height (i.e. 20px wide and 100% height within the leaderboard container)

this is just my understanding of it though, since the stack post I saw gave the solution but like no explanation behind it